tests/kafkatest/tests/streams/streams_upgrade_test.py

```diff
@@ -15,14 +15,16 @@
 
 import random
 import time
-from ducktape.mark import ignore
 from ducktape.mark import matrix
 from ducktape.mark.resource import cluster
 from ducktape.tests.test import Test
+from ducktape.utils.util import wait_until
 from kafkatest.services.kafka import KafkaService
-from kafkatest.services.streams import StreamsSmokeTestDriverService, StreamsSmokeTestJobRunnerService, StreamsUpgradeTestJobRunnerService
+from kafkatest.services.streams import StreamsSmokeTestDriverService, StreamsSmokeTestJobRunnerService, \
+    StreamsUpgradeTestJobRunnerService
 from kafkatest.services.zookeeper import ZookeeperService
-from kafkatest.version import LATEST_0_10_0, LATEST_0_10_1, LATEST_0_10_2, LATEST_0_11_0, LATEST_1_0, LATEST_1_1, LATEST_2_0, DEV_BRANCH, DEV_VERSION, KafkaVersion
+from kafkatest.version import LATEST_0_10_0, LATEST_0_10_1, LATEST_0_10_2, LATEST_0_11_0, LATEST_1_0, LATEST_1_1, \
+    LATEST_2_0, DEV_BRANCH, DEV_VERSION, KafkaVersion
```

Comment:
Seems `LATEST_2_1` is missing here? What about other system tests (maybe it's missing there, too)?

Reply:
No it's not in the system test, but I'm not sure `2.1` is not uploaded to our AWS s3 bucket yet. I'll try to get that loaded, but in the meantime, I'd like this PR to go forward and if need be do a follow-on PR to update the versions in the test (and other system tests) and update `version.py` and `base.sh` at that time as well.  This applies to similar comments below.

Reply:
I've uploaded the new dependencies to AWS so I've added `LATEST_2_1`